### PR TITLE
Refactor prometheus service

### PR DESCRIPTION
**Pull request by YitongFeng**
Refactor prometheus service into several indicidual services

### CODE REVIEW

**Comment by xudifsd** [comment]
this will be conflict with #1282

**Comment by YitongFeng** [comment]
@xudifsd You can merge first, and I solve the conflicts.

File: pai-management/bootstrap/node-exporter/stop.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+INSTANCES="daemonset/node-exporter"
+
+for instance in ${INSTANCES}; do
```
[the remark this answers is not in the document]
Comment (YitongFeng) [reply pai-management/bootstrap/node-exporter/stop.sh]: I think keep it in this way is ok, maybe node-exporter will add other components in the future.

File: pai-management/bootstrap/alert-manager/stop.sh [hunk 2]
```diff
@@ -19,13 +19,7 @@
...
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-INSTANCES="daemonset/node-exporter
-daemonset/watchdog
-deployment/watchdog
-deployment/prometheus-deployment
-configmap/prometheus-configmap
-configmap/prometheus-alert
-configmap/watchdog
+INSTANCES="configmap/prometheus-alert
```
[the remark this answers is not in the document]
Comment (YitongFeng) [reply pai-management/bootstrap/alert-manager/stop.sh]: Resolved

File: pai-management/bootstrap/watch-dog/refresh.sh.template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (YitongFeng) [reply pai-management/bootstrap/watch-dog/refresh.sh.template]: Resolved

**Pull request merged**